typescript-react/package.json

```diff
@@ -97,5 +97,5 @@
     "test:watch": "react-scripts test --env=jsdom"
   },
   "source": "src/index.ts",
-  "version": "1.8.7"
+  "version": "2.0.0"
```

Comment:
1.9.0

Reply:
The idea was to bump the major version since it includes a breaking change in `DslApplication`. Or should it be `1.9.0`? Cc @zrajnis

Reply:
if the argument is optional it should not be a breaking change

Otherwise we would be on 19 already

Reply:
The `1.9.0` change is pushed with --amend.